src/Sulu/Bundle/ContentBundle/Resources/public/js/components/teaser-selection/list/main.js

```diff
@@ -96,6 +96,10 @@ define(['underscore'], function(_) {
             this.sandbox.on('husky.datagrid.teaser-selection.item.deselect', function(id) {
                 this.options.deselectCallback({type: this.options.type, id: id});
             }.bind(this));
+        },
+
+        getUrl: function() {
+            return this.options.url.replace('{locale}', this.options.url);
```

Comment:
when we going this way the `{locale}` need to be added to the content teaser provider. I  would prefer to add here directly `locale=` + this.options.url. So it can't be forgot when a new teaser provider is added. 

Else please update the sulu documentation with the `{locale}` example: http://docs.sulu.io/en/latest/reference/content-types/teaser_selection.html#custom-content-with-teaser-providers